docker/docker-compose.yaml

```diff
@@ -71,20 +71,32 @@ services:
       - CI
       - BINTRAY_USER
       - BINTRAY_KEY
+      - ORG_GRADLE_PROJECT_signingKey
+      - ORG_GRADLE_PROJECT_signingPassword
+      - SONATYPE_USER
+      - SONATYPE_TOKEN
     command: >
       bash -cl "./gradlew --no-daemon --parallel --max-workers=4 clean check &&
       ./gradlew --no-daemon bintrayUpload"
 
+# Don't run `publish` task on `publish-snapshot` for now ^^ replace with below command later.
+#      bash -cl "./gradlew --no-daemon --parallel --max-workers=4 clean check &&
+#      ./gradlew --no-daemon publish bintrayUpload"
```

Comment:
You can uncomment this now. We will test that everything works with staging repo. When we are ready to merge this PR we can start publishing snapshots to sonatype.

Reply:
Yes that's what I had in mind